statsmodels/base/model.py

```diff
@@ -821,17 +817,16 @@ def predict(self, exog=None, transform=True, *args, **kwargs):
                 exog = exog[:, None]
             exog = np.atleast_2d(exog)  # needed in count model shape[1]
 
-        predict_results = self.model.predict(self.params, exog, *args, **kwargs)
-
-        if exog_index is not None and not hasattr(predict_results, 'predicted_values'):
+        predict_results = self.model.predict(self.params, exog, *args,
+                                             **kwargs)
 
+        if exog_index is not None and not hasattr(predict_results,
+                                                  'predicted_values'):
```

Comment:
do you remember or know why we have the hasattr predicted_values condition?
I don't remember which model might return this.

Reply:
Can't say that I do...